README.md

```diff
@@ -15,45 +15,49 @@ export AWS_SECRET_ACCESS_KEY="YOUR_SECRET_KEY"
 export AWS_REGION="us-east-1"
 ```
 
+You should write them into the bashrc-file as well, so that they are loaded each time you open a terminal.
+```
+vim ~/.bashrc
+```
 Then you will be able to pull the docker images using the following command:
 
 ```
 docker-compose up
 ```
-Then start the people-api using your IDE or this command
+Then start the people-api using your IDE or the following command. This should also work if `docker-compose up` failed.
 ```
 ./gradlew bootRun
 ```
 
 You will be able to see the people-api service registered in eureka here:
 ```
-http://localhost:8761/
+http://localhost:8081/
 ```
 
 And you can access the people-api and it's swagger here: 
 ```
-http://localhost:9081/people-service/people
-http://localhost:9081/people-service/swagger-ui.html
+http://localhost:8081/people
+http://localhost:8081/swagger-ui.html
```

Comment:
These links also depend on weather the edge server are running or not.
If it's running you can access the api with both links. With the old ones you are accessing the api through the edge server and with the ones you provided you access the api directly. You can always do this, doesn't matter if edge is running or not.

Reply:
so should I include both? or just leave it as it was?